### ISSUE TITLE
Doc : User guide 

### ISSUE THREAD

**Issue by jpoullet2000** [taxpon/openpyscad#7]
What about a user guide doc? 
I thought it might be somewhat sexier ;)...  and easier to get some insight about the possibilities of the package +  could be quite useful as we keep adding new functionalities.

**Comment by taxpon** [comment]
@jpoullet2000 I totally agree with your idea. But I am wondering that current README is different from the user guide in your sense? Is the user guide in your sense a document which is focus on pros/cons of this package in comparison to other OpenScad Python package (like SolidPython) including usage of summary of this package?

**Comment by jpoullet2000** [comment]
By doc, I was thinking of a gallery of examples. This gallery would follow two goals: showing concretely the potential of the tool and making some comparison with openscad (in terms of syntax) + maybe other openscad python packages. I would not compare too much with other openscad Python packages.... 
So wrt to the README I would essentially add pictures  (which can easily be generated with openscad tool). About style/templating I would put it on readthedocs... I can create the account if you wish ;)...
